services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/grpc/LogicalPortSpec.groovy

```diff
@@ -0,0 +1,78 @@
+package org.openkilda.functionaltests.spec.grpc
+
+
+import static org.junit.Assert.assertTrue
+import static org.openkilda.testing.ConstantsGrpc.LOGICAL_PORT
+import static org.openkilda.testing.ConstantsGrpc.SWITCH
+import static org.openkilda.testing.ConstantsGrpc.SWITCH_PORT
+
+import org.openkilda.functionaltests.BaseSpecification
+import org.openkilda.grpc.speaker.model.LogicalPortDto
+import org.openkilda.messaging.error.MessageError
+
+import groovy.util.logging.Slf4j
+import org.springframework.http.HttpStatus
+import org.springframework.web.client.HttpClientErrorException
+import spock.lang.Narrative
+import spock.lang.Unroll
+
+@Slf4j
+@Narrative("Test Logical Port.")
+class LogicalPortSpec extends BaseSpecification {
+    @Unroll
+    def "Not able to create logical port with incorrect ports number(lPort/sPort): #logicalPortNumber/#portNumber."() {
+        when:
+        " Try to create with incorrect values (logical port/simple port): $logicalPortNumber/$portNumber."
```

Comment:
use hash instead of 'dollar': `$logicalPortNumber` -> `#logicalPortNumber`
This is spock-specific behavior, not just a string. To form a valid report you should use the hash option